packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/http_response_error.dart

```diff
@@ -38,8 +40,12 @@ class HttpResponseError {
   /// Used by the platform implementation to create a new [HttpResponseError].
   const HttpResponseError({
```

Comment:
I think we also want a `WebResourceResponse?` class to differentiate between the request and response values. Specifically headers.